src/languages/fsharp.js

```diff
@@ -127,20 +123,47 @@ export default function(hljs) {
         ]
       },
       {
-        className: 'meta',
-        begin: '\\[<',
-        end: '>\\]',
-        relevance: 10
+        // computation expressions:
+        beginScope: "emphasis",
```

Comment:
No, this is a `markup` scope... it's not allowed in code definition.

Reply:
Ah that's too bad, I liked the subtle difference!
(it's hard to pick a scope, it does not seem to completely fit in any existing one... I think I'll go for `keyword`, even though it's possible to define custom computation expressions)

Reply:
What is it called exactly in F#? What is the closed equivalent in other languages?

Reply:
It's called a [computation expression](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/computation-expressions). I don't know of an equivalent in other languages.

They are F#'s way to implement syntactic sugar for monads, and they also can be used to create simple but powerful DSL.
For example, they are used to implement async workflows:
```fsharp
let work =
    async { // async computation expression
        let delayTask () =
            task { // task computation expression (compatible with C# Tasks)
                printfn "Delay..."
                // bang keywords can be used inside computation expressions.
                // The semantic of this depends on its implementation.
                do! Task.Delay 1000
                return 42
            }
        let! result = delayTask () |> Async.AwaitTask
        printfn "Async F# sleep..."
        do! Async.Sleep 1000
        return result
    }

let result = work |> Async.RunSynchronously
```

As another and completely different example, they are used to create a DSL to initialize a web application with the Saturn F# web framework: https://saturnframework.org/reference/Saturn/saturn-application-applicationbuilder.html
```fsharp
let app = application {
    pipe_through endpointPipe
    use_router topRouter
    url "http://0.0.0.0:8085/"
    memory_cache
    use_static "static"
    use_gzip
}
```

The previous `application` computation expression defines several new keywords that can be used in the computation (`pipe_through`, `use_router`, `url`...)

Reply:
Wouldn't `{}` be punctuation (or operators) and the name perhaps `title.function`?  If someone can define DSLs with them like this then it sounds a lot like a type of function/proc/lambda, etc...

Reply:
Semantically it has nothing to do with a function (it does not define something callable in any way).
Do you mean syntactically maybe?

I feel like it would fit better as a keyword (a keyword before a block that can contain statements from a DSL). 
It seems other code highlighting libraries supporting F# interpret it that way.

Reply:
> Do you mean syntactically maybe?

Yes.

I'm uncertain.  Lets use [classNameAliases](https://highlightjs.readthedocs.io/en/latest/mode-reference.html#classnamealiases) to give it the proper F# name/scope `computation-expression.`, and then alias it back down to keyword for now.

Curious, do you know what scope TextMate/VSCode applies to these?

Reply:
> Curious, do you know what scope TextMate/VSCode applies to these?

No sorry, I only checked prismJS and github...